import-service/serverless.yml

```diff
@@ -17,6 +19,7 @@ functions:
       - http:
           path: import
           method: get
+          authorizer: arn:aws:lambda:eu-west-1:009688690949:function:authorization-service-dev-basicAuthorizer
```

Comment:
As we implement automation in creating instance it is better to avoid hardcoded refs like this. You can use import/export through services. In that case you avoid side effects if for any reason this link will change

Reply:
Thanks!